### PR TITLE
feat/boltz fee info

### PR DESCRIPTION
**Pull request by dni**
- improve fee information an docs
- review @michael1011
- update boltz_client to 0.1.3
- add boltz fee dynamically

### CODE REVIEW

File: lnbits/extensions/boltz/templates/boltz/_api_docs.html [hunk 1]
```diff
@@ -78,7 +79,9 @@ <h3 class="text-subtitle1 q-my-none">
         <li>Onchain lockup tx fee: whatever you choose when paying</li>
         <li>Onchain claim tx fee: ~500 sats</li>
         <li>Routing fees (paid by boltz): unknown</li>
-        <li>Boltz fees: 500 sats (0.5%)</li>
+        {% raw %}
+        <li>Boltz fees: 500 sats ({{ boltzConfig.fee_percentage }}%)</li>
```
Comment (callebtc) [inline lnbits/extensions/boltz/templates/boltz/_api_docs.html:83]: Why is 500 fixed but the fee percentage can change?

Comment (michael1011) [reply lnbits/extensions/boltz/templates/boltz/_api_docs.html:83]: I agree completely. This should be calculated dynamically too!

Comment (dni) [reply lnbits/extensions/boltz/templates/boltz/_api_docs.html]: fixed it :)

File: lnbits/extensions/boltz/templates/boltz/_api_docs.html [hunk 2]
```diff
@@ -73,21 +83,30 @@ <h3 class="text-subtitle1 q-my-none">
...
+        </li>
+        <li>
+          You pay onchain: {{ boltzExample.normal_expected_amount }} sats +
+          lockup_fee
+        </li>
+        <li>You receive lightning: {{ boltzExample.amount }} sats</li>
       </ul>
       <p>
         onchain_payment + lockup_fee = amount + (amount * boltz_fee / 100) +
         claim_fee + lockup_fee
```
Comment (michael1011) [inline lnbits/extensions/boltz/templates/boltz/_api_docs.html:107]: I'd remove the `+ lockup_fee` from both sides

**Review by michael1011 — APPROVED** [review]

File: lnbits/extensions/boltz/templates/boltz/_api_docs.html (hunk 2 above)
Comment (dni) [reply lnbits/extensions/boltz/templates/boltz/_api_docs.html]: 1c3a9d18

**Review by michael1011 — APPROVED** [review]

**Pull request merged**